docs/reference/overview_of_strategies.rst

```diff
@@ -250,7 +250,91 @@ This strategy plays randomly (disregarding the history of play).
 Axelrod's second tournament
 ---------------------------
 
-Work in progress.
+The code for Axelrod's second touranment was originally published by the
+`University of Michigan Center for the Study of Complex Systems <http://lsa.umich.edu/cscs/>`_
+and is now available from
+`Robert Axelrod's personal website <http://www-personal.umich.edu/~axe/research/Software/CC/CC2.html>`_.
+
+The Axelrod-Python organisation has published a
+`modified version of the original code <https://github.com/Axelrod-Python/TourExec>`_.
+In the following table, links to original code point to the Axelrod-Python
+repository.
+
+.. include:: fortan_keys.rst
+
+.. csv-table:: Strategies in Axelrod's second tournament
+   :header: "Original Code", "Name", "Axelrod Library Name"
+   :widths: 27, 24, 60
+
+   "GRASR_", "Grasr", "Not Implemented"
+   "K31R_", "Grisell", "Not Implemented"
+   "K32R_", "Kluepfel", "Not Implemented"
+   "K33R_", "Rabbie", "Not Implemented"
+   "K34R_", "Friedman", "Not Implemented"
+   "K35R_", "Getzler", "Not Implemented"
+   "K36R_", "Hotz", "Not Implemented"
+   "K37R_", "Lefevre", "Not Implemented"
+   "K38R_", "Weiderman", "Not Implemented"
+   "K39R_", "Almy", "Not Implemented"
+   "K40R_", "Adams", "Not Implemented"
+   "K41R_", "Weiner", "Not Implemented"
+   "K42R_", "Borufsen", "Not Implemented"
+   "K43R_", "Anderson", "Not Implemented"
+   "K44R_", "Adams", "Not Implemented"
+   "K45R_", "Mcgurrin", "Not Implemented"
+   "K46R_", "Eatherley", ":class:`axelrod.strategies.axelrod_second.Eatherley`"
+   "K47R_", "Hufford", "Not Implemented"
```

Comment:
Yep, there are some authors who submitted more than one strategy. I'm just listing the name here at the moment and not attempting to make it unique in any way.